### PR TITLE
AB#40 Add log information in command service application

### PR DESCRIPTION
**Pull request by pilo1987**

### CODE REVIEW

File: cqrs-user-command-service/src/main/java/dev/amir/usercommand/UserCommandServiceApplication.java [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+@Slf4j
 @SpringBootApplication
 public class UserCommandServiceApplication {
 
     public static void main(String[] args) {
+        log.info("Starting User Command Service application.");
```
Comment (Ramir0) [inline cqrs-user-command-service/src/main/java/dev/amir/usercommand/UserCommandServiceApplication.java:12]: this log is not part of the application layer (application folder in the UserCommandService project)

**Review by Ramir0 — CHANGES_REQUESTED** [review]

File: cqrs-user-command-service/src/main/java/dev/amir/usercommand/UserCommandServiceApplication.java (hunk 1 above)
Comment (pilo1987) [reply cqrs-user-command-service/src/main/java/dev/amir/usercommand/UserCommandServiceApplication.java]: "The required changes have been resolved."

File: cqrs-user-command-service/src/main/java/dev/amir/usercommand/application/usecase/UserUseCases.java [hunk 2]
```diff
@@ -17,31 +19,37 @@ public class UserUseCases implements UserInputPort {
...
 
     @Override
     public void updateUser(User user) {
+        log.info("Verifying if the 'id' field exists for user update.");
         if (!StringUtils.hasText(user.getId())) {
             throw new IllegalArgumentException("Invalid User, id field must exist");
         }
 
         User savedUser = userOutputPort.save(user);
+        log.info("User successfully created/updated with id: " + savedUser.getId());
```
Comment (Ramir0) [inline cqrs-user-command-service/src/main/java/dev/amir/usercommand/application/usecase/UserUseCases.java:41]:
```suggestion
        log.info("User successfully updated with id: " + savedUser.getId());
```

**Review by Ramir0 — CHANGES_REQUESTED** [review]

File: cqrs-user-command-service/src/main/java/dev/amir/usercommand/application/usecase/UserUseCases.java [hunk 3]
```diff
@@ -17,31 +19,37 @@ public class UserUseCases implements UserInputPort {
...
 
     @Override
     public boolean deleteUser(String userId) {
+        log.info("Attempting to delete user with ID: " + userId);
         if (!StringUtils.hasText(userId)) {
             throw new IllegalArgumentException("Invalid User, id field must exist");
         }
-
+        
+        log.info("Attempting to delete user with ID: " + userId);
```
Comment (Ramir0) [inline cqrs-user-command-service/src/main/java/dev/amir/usercommand/application/usecase/UserUseCases.java:52]: this line is duplicated from line 47, change the message

**Review by Ramir0 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**